resolwe_bio/processes/amplicon/bwa_trim.yml

```diff
@@ -0,0 +1,180 @@
+# ==========================================
+# Align reads and trimm resulting SAM output
+# ==========================================
+---
+
+- slug: align-bwa-trim
```

Comment:
It trims primers after aligment directly from SAM file, only than the file is converted to BAM.